src/views/docs-view/server.ts

```diff
@@ -345,11 +345,30 @@ export function getStaticGenerationFunctions<
 				layoutProps.githubFileUrl = githubFileUrl
 			}
 
+			/**
+			 * TODO: this logic is temporary to showcase how layouts can work.
+			 * Going forward this would be defined in frontmatter,
+			 * we'd pass frontmatter.layout directly rather than constructing this.
+			 *
+			 * Task: https://app.asana.com/0/1202097197789424/1202850056121889/f
```

Comment:
Grouped the `layout` shim together a bit here, and added an Asana task for further `layout` work follow-up. Other than that, this is effectively identical to what was in #733.